src/SqlSerializer.h

```diff
@@ -74,25 +75,26 @@ class SqlSerializer {
 
     sqlite3_stmt *populate_insert_argument_statement(const closure_info_t &info,
                                                      int index);
+
     bool verbose;
     int indentation;
-    sqlite3 *database = nullptr;
+    sqlite3 *database = NULL;
     std::ofstream trace;
-    sqlite3_stmt *insert_metadata_statement = nullptr;
-    sqlite3_stmt *insert_function_statement = nullptr;
-    sqlite3_stmt *insert_argument_statement = nullptr;
-    sqlite3_stmt *insert_call_statement = nullptr;
-    sqlite3_stmt *insert_promise_statement = nullptr;
-    sqlite3_stmt *insert_promise_association_statement = nullptr;
-    sqlite3_stmt *insert_promise_evaluation_statement = nullptr;
-    sqlite3_stmt *insert_promise_return_statement = nullptr;
-    sqlite3_stmt *insert_promise_lifecycle_statement = nullptr;
-    sqlite3_stmt *insert_promise_argument_type_statement = nullptr;
-    sqlite3_stmt *insert_gc_trigger_statement = nullptr;
-    sqlite3_stmt *insert_type_distribution_statement = nullptr;
-    sqlite3_stmt *insert_environment_statement = nullptr;
-    sqlite3_stmt *insert_variable_statement = nullptr;
-    sqlite3_stmt *insert_variable_action_statement = nullptr;
+    sqlite3_stmt *insert_metadata_statement = NULL;
```

Comment:
I got this form Peta, who said that using nullptr is preferable to NULL since nullptr is type checked. Not sure why you changed it.